api/src/main/java/org/hawkular/inventory/api/Inventory.java

```diff
@@ -26,4 +34,83 @@
 public interface Inventory extends AutoCloseable {
 
     Tenants.ReadWrite tenants();
+
+    /**
+     * Quickly navigate to the provided tenant.
+     *
+     * @param tenant the tenant to steer to.
+     * @return the access interface to the tenant
+     * @throws EntityNotFoundException if the tenant is not found
+     */
+    default Tenants.Single steerTo(Tenant tenant) throws EntityNotFoundException {
```

Comment:
I agree that these should really be out of the `Inventory` interface. Now we just need to find a good name.

I chose "steer" or "navigate" because to me that stresses the act of quering more than the actual retrieval of the entities (which "find" or "look up" would).

what about:
```
Inspect.in(inventory).environment(env)...
```

inspecting something in inventory doesn't evoke state change nor retrieval, IMHO...